### DISCUSSION TITLE
The piano song "Your Reality" is modified

### DISCUSSION THREAD

**Discussion by srfoxapp** [Monika-After-Story/MonikaModDev#8744] [QA - tech support + general] [answered]
In the line "The ink flows down into a dark puddle", monika sweats as if she doesn't recognize the line, is it some uptade?

https://user-images.githubusercontent.com/99615091/153772025-39f6e74b-8110-4258-afa1-c8015c60defd.mp4

**Comment by ThePotatoGuy** [comment, marked as answer]
it was changed 2 versions ago, see the [notes under Gameplay Changes](https://github.com/Monika-After-Story/MonikaModDev/releases/tag/v0.12.6)

**Comment by JinSakaiM** [comment]
thanks for the tutorial!
